### PR TITLE
third commit

### PR DESCRIPTION
**Pull request by appaajee**
I added a new package and a class in it

### CODE REVIEW

File: aj/src/ajpackage/test.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package ajpackage;
+
+public class test {
+	static int a=5;
+	static int b=10;
+	static int z = a+b;
+	public static void main(String[] args) {
```
Comment (ajitajee) [inline aj/src/ajpackage/test.java:4]: why don't you write all variables in main code

Comment (ajitajee) [inline aj/src/ajpackage/test.java:4]: dfs

**Comment by ajitajee** [comment]
see the review

**Comment by appaajee** [comment]
ok

**Comment by ajitajee** [comment]
go ahead and merge

**Pull request merged**